lib/recurly/hosted-fields.js

```diff
@@ -109,17 +112,38 @@ export class HostedFields extends Emitter {
     this.initQueue = [];
   }
 
-  readyHandler (body) {
+  onReady (body) {
     const pos = this.initQueue.indexOf(body.type);
     if (~pos) this.initQueue.splice(pos, 1);
     if (this.initQueue.length === 0) {
-      this.off('hostedField:ready', this.readyHandler);
+      this.off('hostedField:ready', this.onReady);
       this.bus.send('hostedFields:ready');
       this.ready = true;
     }
     this.update(body);
   }
 
+  onTab (direction, message) {
+    const origin = this.getFieldByType(message.type);
+    let tabbableItems = this.tabbableItems();
+    let destination;
+
+    // Find the origin within the tabbable list, and focus in our intended direction
+    for (let i = 0; i < tabbableItems.length; i++) {
+      if (origin instanceof HostedField) {
+        if (origin.tabbingProxy == tabbableItems[i]) {
+          destination = direction == 'previous' ? tabbableItems[i - 1] : tabbableItems[i + 1];
+        }
+      }
+    }
+
+    if (destination) destination.focus();
+  }
+
+  tabbableItems () {
```

Comment:
@chrissrogers : I had to wrap the tabbable call in order to properly spy on it. See below:

https://github.com/sinonjs/sinon/issues/664